rpkg_src/extract_temp_from.cpp

```diff
@@ -1,4 +1,5 @@
 #include "rpkg_function.h"
+/*
```

Comment:
The function here is empty.

Reply:
From memory, it was used in the past and removed as it was render useless. But the function may still be refereced elsewhere hence why it wasn't removed.

Reply:
It's an empty function. Might as well do it.

Reply:
Done.